compiler/coreSyn/CoreSubst.hs

```diff
@@ -479,12 +479,17 @@ substIdBndr _doc rec_subst subst@(Subst in_scope env tvs cvs) old_id
     (Subst (in_scope `extendInScopeSet` new_id) new_env tvs cvs, new_id)
   where
     id1 = uniqAway in_scope old_id      -- id1 is cloned if necessary
+    -- Why is this not substIdType???
```

Comment:
Do you intend this comment to stay?

Reply:
Yes, it's a question that deserves to be preserved - https://ghc.haskell.org/trac/ghc/ticket/15345#comment:1

Reply:
Then: merge when green.